app/docs/enterprise/0.31-x/developer-portal/introduction.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Welcome to the Kong Developer Portal
+title: Welcome to the Kong Developer Portal Preview
 ---
 
-# [Welcome to the Kong Dev Portal](#welcome-to-the-kong-dev-portal)
+# [Welcome to the Kong Dev Portal Preview](#welcome-to-the-kong-dev-portal)
 
 ![alt text](https://konghq.com/wp-content/uploads/2018/03/screen-home.png "Welcome to the Kong Dev Portal")
 
-Thanks for installing or upgrading Kong Enterprise Edition. This document orients you to Kong's built-in Dev Portal functionality and gets you started with customizing your Kong Dev Portal.
+Thank you for installing Kong Enterprise Edition. This document orients you to Kong's built-in Dev Portal functionality and provides you a preview into it so you can familiarize yourself with its promising capabilities and get started on customizing it to match your needs.
```

Comment:
Suggestion: "and provides you a preview ~~into it~~ so you can familiarize yourself"